docs/api/algorand/many-to-one-transfer.mdx

```diff
@@ -0,0 +1,94 @@
+# ManyToOneTransfer
+
+Send funds from multiple [DelegatedWallet](#TODO) (owned by the same account), to another (regular or delegated) wallet.
+
+## Request Parameters
+
+|     Name    |                    Type                   | Description |
+| ----------- | ----------------------------------------- | ----------- | 
+| `assetId`   | `string`                                  | The ID of the DeCash asset that is being sent. |
+| `senders`   | `Array<{address:string, amount:number}>`  | Array of objects containing the `address` of the accounts that are sending the funds, and the respective `amount` that are being sent. Note that each account must have enough funds to cover both the `amount`, and the [fee](#TODO). |
+| `receiver`  | `string`                                  | The address of the account that is receiving the funds. Note that, if this account is not a DelegatedWallet, then it must be opted into the token that is being sent. | 
+
+## Responses
+
+### Regular response
+
+|     Name    |         Type        | Description |
+| ----------- | ------------------- | ----------- | 
+| `noSign`    | `string[]`          | The array of transactions (in msgpack) that shall not be signed. Together with `toSign` it composes the full array of transactions in the group. |
+| `toSign`    | `string[]`          | The array of transactions (in msgpack) that must be signed with the spending key of the `owner` of the DelegatedWallets. Together with the `noSign` transactions it composes the full array of transactions in the group. |
+| `info`      | `{fee:number}`      | An object containing `fee`, the amount of DeCash that is being spent on fees in total in this transaction. (e.g. 1.05 USDD) | 
+
+### Validation errors
+
+Returned when the parameters are invalid.
+
+|     Name    |                              Content                              |
+| ----------- | ----------------------------------------------------------------- |
+| Status code | 400                                                               |
+| `errorId`   | `INVALID_PARAMS`                                                  |
+| `message`   | `Validation errors in ManyToOneTransfer request.`                 | 
```

Comment:
if it's specifically a string, use `"string quotes inside the codeblock"`. Same for the rest of the doc